From 1f7777eb6034708bc4a74cf0092c99927d7044eb Mon Sep 17 00:00:00 2001
From: Green Wan <green.wan@sifive.com>
Date: Mon, 18 Jan 2021 00:28:45 -0800
Subject: [PATCH 28/52] sifive: hifive_unmatched_fu740: Enable 64bit PCI
 resources

Enable 64bit PCI resources support to avoid address is truncated in
new u-boot code.

Signed-off-by: Green Wan <green.wan@sifive.com>
---
 include/configs/sifive-hifive-unmatched-fu740.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/sifive-hifive-unmatched-fu740.h b/include/configs/sifive-hifive-unmatched-fu740.h
index 34d264761d..30458d2b66 100644
--- a/include/configs/sifive-hifive-unmatched-fu740.h
+++ b/include/configs/sifive-hifive-unmatched-fu740.h
@@ -36,6 +36,8 @@
 
 #define CONFIG_STANDALONE_LOAD_ADDR	0x80200000
 
+#define CONFIG_SYS_PCI_64BIT		1	/* enable 64-bit resources */
+
 /* Environment options */
 
 #ifndef CONFIG_SPL_BUILD
-- 
2.30.2

